raiden/transfer/node.py

```diff
@@ -327,33 +331,27 @@ def handle_init_unlock_light(
 def handle_init_send_lock_expired_light(
     chain_state: ChainState, state_change: ActionSendLockExpiredLight
 ) -> TransitionResult[ChainState]:
-    channel_state = views.get_channelstate_by_canonical_identifier_and_address(
-        chain_state=chain_state,
-        canonical_identifier=CanonicalIdentifier(
-            chain_identifier=chain_state.chain_id,
-            token_network_address=state_change.signed_lock_expired.token_network_address,
-            channel_identifier=state_change.signed_lock_expired.channel_identifier,
-        ),
-        address=state_change.sender
-    )
-
```

Comment:
could you briefly explain why this isn't needed anymore?

Reply:
We removed the receiver_light_client_address since we are not using both columns anymore so, we were using channel_state to get that, but now we don't so we don't need to get the channel_state anymore